client/components/app.jsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import ReviewList from './reviewList.jsx';
+import PageSelector from './pageSelector.jsx';
+
+class App extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      allPreviews: props.reviews,
+      overallStar: 3.75,
+      currPage: 1
+    }
+
+  }
+
+  render() {
+    let reviews = this.state.allPreviews.slice((this.state.currPage - 1) * 5, this.state.currPage * 5);
```

Comment:
Do the reviews match up with the specific experience_id that the user is viewing?